crates/rv/src/commands/ruby/run.rs

```diff
@@ -36,19 +36,25 @@ pub(crate) enum CaptureOutput {
 /// `CaptureOutput::No`, this returns an empty `Output` struct.
 pub(crate) async fn run<A: AsRef<std::ffi::OsStr>>(
     config: &Config,
-    request: &RubyRequest,
+    version: Option<RubyRequest>,
```

Comment:
I think we should document in the docstring above (line 36) what happens if `version` is `None`.